fns-hl7-pipeline/fn-redactor/host.json

```diff
@@ -3,7 +3,18 @@
   "extensionBundle": {
     "id": "Microsoft.Azure.Functions.ExtensionBundle",
     "version": "[4.0.0, 4.3)"
-  }, 
+  },
```

Comment:
Instead of removing it, can we just tune the values
use maxEventBatchSize: 10, Prefetch: 20?

Reply:
never mind